baseapp/baseapp.go

```diff
@@ -81,6 +75,7 @@ type BaseApp struct {
 	postHandler sdk.PostHandler // post handler, optional, e.g. for tips
 
 	initChainer          sdk.InitChainer                // ABCI InitChain handler
+	preBeginBlocker      sdk.PreBeginBlocker            // logic to run before BeginBlocker
```

Comment:
thoughts on calling it preblocker?

Reply:
sounds good to me, the shorter the better.